code/script.js

```diff
@@ -2,10 +2,16 @@
 const board = document.getElementById('board')
 const questions = document.getElementById('questions')
 const restartButton = document.getElementById('restart')
+const findOutButton =  document.getElementById('filter')
```

Comment:
Think you just forgot to remove this since it seems like you removed the corresponding tags from the html doc (not in use)